<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Analysis results</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="analTable"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="infoAnalButt">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Information about analysis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buildGraphButt">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Build Graph</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeDBButt">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Close analysis</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_analysisi"/>
    <addaction name="actionOpen_analysis"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuFilter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_analysisi">
   <property name="text">
    <string>New analysisi</string>
   </property>
  </action>
  <action name="actionOpen_analysis">
   <property name="text">
    <string>Open analysis</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
